### PR TITLE
Create bootstrap-helm image

### PR DESCRIPTION
**Pull request by jakkab**
**Description**

Changes proposed in this pull request:

- add testing.sh to the GKE integration pipeline
- create bootstrap-based GKE integration image with helm command installed

**Related issue(s)**
#139
#20 - continued

### CODE REVIEW

File: prow/images/gke-integration/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM eu.gcr.io/kyma-project/prow/bootstrap:0.0.1
+
+ENV HELM_VERSION="v2.10.0"
+
+RUN wget -q https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz -O - | tar -xzO linux-amd64/helm > /usr/local/bin/helm \
+    && chmod +x /usr/local/bin/helm
```
Comment (Demonsthere) [inline prow/images/gke-integration/Dockerfile]: I think we also need to initialise helm in the docker image: `helm init --client-only`

Comment (jakkab) [reply prow/images/gke-integration/Dockerfile]: fixed!

File: prow/images/gke-integration/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM eu.gcr.io/kyma-project/prow/bootstrap:0.0.1
+
+ENV HELM_VERSION="v2.10.0"
+
```
Comment (aszecowka) [inline prow/images/gke-integration/Dockerfile]: please create a job: https://github.com/kyma-project/test-infra/blob/master/prow/jobs/test-infra/buildpack.yaml

File: prow/jobs/kyma/kyma-integration.yaml [hunk 3]
```diff
@@ -45,7 +45,7 @@ presubmits: # runs on PRs
           path_alias: github.com/kyma-project/test-infra
       spec:
         containers:
-          - image: eu.gcr.io/kyma-project/prow/bootstrap:0.0.1
+          - image: eu.gcr.io/kyma-project/prow/gke-integration:0.0.1
```
Comment (aszecowka) [inline prow/jobs/kyma/kyma-integration.yaml]: regarding image name, I also don't have a good idea how to name it, but maybe bootstrap-helm will be a better option?

**Comment by jakkab** [comment]
@aszecowka Comments addressed, please review.

**Pull request merged**